metadrive/manager/replay_manager.py

```diff
@@ -16,7 +16,7 @@
 
 
 class ReplayManager(BaseManager):
-    PRIORITY = 100  # lowest
+    PRIORITY = 99  # lowest
```

Comment:
PRIORITY will affect its order in engine.managers list? Larger number the latter?

Reply:
The larger the lower. ReplayManager should slightly higher than RecordManager. This is to ensure that RecordManager can be run after all managers

Reply:
While both record and replay manager won't work together actually